sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_call_connection.py

```diff
@@ -49,49 +81,69 @@ def hang_up(
 
     @distributed_trace()
     def cancel_all_media_operations(
-            self,
-            operation_context=None,  # type: Optional[str]
-            **kwargs  # type: Any
-        ): # type: (...) -> CancelAllMediaOperationsResult
+        self,
+        operation_context=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ): # type: (...) -> CancelAllMediaOperationsResult
 
         cancel_all_media_operations_request = CancelAllMediaOperationsConverter.convert(operation_context)
 
-        return self._call_connection_client.cancel_all_media_operations(
+        cancel_all_media_operations_result = self._call_connection_client.cancel_all_media_operations(
```

Comment:
why we need to use ._from_generated to cast CancelAllMediaOperationsResult to a customized type. 
For Python, we can return type from generated _models.py or _models3.py directly